spec/repository_spec.rb

```diff
@@ -42,13 +44,55 @@
 
     expect(chef_run.remote_file('remote_file_DATADOG_APT_KEY_CURRENT.public')).to notify(
       'execute[import apt datadog key DATADOG_APT_KEY_CURRENT.public]').to(:run).immediately
+    expect(chef_run.remote_file('remote_file_5F1E256061D813B125E156E8E6266D4AC0962C7D')).to notify(
+      'execute[import apt datadog key 5F1E256061D813B125E156E8E6266D4AC0962C7D]').to(:run).immediately
     expect(chef_run.remote_file('remote_file_D75CEA17048B9ACBF186794B32637D44F14F620E')).to notify(
       'execute[import apt datadog key D75CEA17048B9ACBF186794B32637D44F14F620E]').to(:run).immediately
     expect(chef_run.remote_file('remote_file_A2923DFF56EDA6E76E55E492D3A80E30382E94DE')).to notify(
       'execute[import apt datadog key A2923DFF56EDA6E76E55E492D3A80E30382E94DE]').to(:run).immediately
   end
 end
 
+def handle_keys(key_list, install_gnupg = true)
```

Comment:
I don't feel like `handle_keys` is a very descriptive method name. Can you please improve it?

Reply:
I proposed `import_gpg_keys`, wdyt?